### ISSUE TITLE
More QA checks suggestions

### ISSUE THREAD

**Issue by rubickcz** [druids/developers-chamber#58]
- unsused imports (probably need to check all code, not only changes)
- ~~invalid test method names (`test_should_` is forbidden)~~ (implemented in #66)

**Comment by radimsuckr** [comment]
Test method naming check should be configurable (regex pattern or something like that).

**Comment by rubickcz** [comment]
@radimsuckr good point, however we just merged it hardcoded. I've discovered your comment late. But nevermind, we can implement this later.
